src/pages/workspaces/workspace/Tools.js

```diff
@@ -18,22 +20,63 @@ const styles = {
     padding: '1rem 4rem',
     display: 'flex', flexWrap: 'wrap'
   },
-  card: {
+  shortCard: {
     ...Style.elements.card, width: 300, height: 125, margin: '1rem 0.5rem',
     display: 'flex', flexDirection: 'column', justifyContent: 'space-between'
   },
   shortTitle: {
-    flex: 'none',
+    flex: 1,
     color: colors.blue[0], fontSize: 16,
     lineHeight: '20px', height: '40px',
     overflow: 'hidden', overflowWrap: 'break-word'
-  }
+  },
+  shortDescription: {
+    flex: 'none',
+    lineHeight: '18px', height: '90px',
+    overflow: 'hidden'
+  },
+  longCard: {
+    ...Style.elements.card,
+    width: '100%', minWidth: 0,
+    margin: '0.25rem 0.5rem'
+  },
+  longTitle: {
+    color: colors.blue[0], fontSize: 16,
+    whiteSpace: 'nowrap', overflow: 'hidden', textOverflow: 'ellipsis', flex: 1
+  },
+  longDescription: {
+    flex: 1,
+    paddingRight: '1rem',
+    whiteSpace: 'nowrap', overflow: 'hidden', textOverflow: 'ellipsis'
+  },
+  toolbarContainer: {
```

Comment:
note for onlookers: Normally we would try to share code with the other instances of this, but there are major visual changes happening in another PR, so we're copying for now and will unify them after.

Reply:
This is a fantastic PR comment.